src/components/marker-chart/index.js

```diff
@@ -6,6 +6,8 @@
 import * as React from 'react';
 import explicitConnect from '../../utils/connect';
 import MarkerChartCanvas from './Canvas';
+import EmptyReasons from './EmptyReasons';
```

Comment:
I find this confusing and would prefer it to be named `MarkerChartEmptyReasons`. That'll make the file and the component definition easier to find.

Reply:
(If you also rename the file, I mean.)

Reply:
I guess you have the same comment for the call tree's component too ?

Reply:
I suppose so, yes.